grpc/pb-marshalling.go

```diff
@@ -71,14 +71,12 @@ func ChallengeToPB(challenge core.Challenge) (*corepb.Challenge, error) {
 		}
 	}
 
-	var validatedInt int64 = 0
-	validatedTS := timestamppb.New(time.Time{})
+	validated := timestamppb.New(time.Time{})
```

Comment:
Contrary to the new guidance in CONTRIBUTING.md, this code is fully capable of sending a populated-but-zero `validated` timestamp. Instead of creating a new timestamppb holding the Go zero time, this should say
```suggestion
	var validated *timestamppb.Timestamp
```
so that it remains `nil` if it doesn't get populated by the conditional on the next line.

The same is true in a bunch of similar situations below.

Reply:
I've fixed every occurrence of this.